lib/erl_interface/src/prog/erl_call.c

```diff
@@ -989,3 +1023,87 @@ static char* ei_chk_strdup(char *s)
     }
     return p;
 }
+
+/*
+ * Helper function that that:
+ *
+ * 1. Executes a function on a remote node
+ *
+ * 2. Forwards what the executed function and its subprocesses prints
+ *    to stdout of this process
+ *
+ * 3. Returns the result of the executed function (the result term is
+ *    written to the buffer pointed to by x)
+ *
+ * This function is similar to (and is based on) the function ei_rpc
+ */
+static int rpc_print_node_stdout(ei_cnode* ec, int fd, char *mod,
+                                 char *fun, const char* inbuf,
+                                 int inbuflen, ei_x_buff* x)
+{
+    int i, index;
+    int got_rex_response = 0;
+    int initial_buff_index = x->index;
+    ei_term t;
+    erlang_msg msg;
+    char rex[MAXATOMLEN];
+ 
+    if (ei_xrpc_to(ec, fd, mod, fun, inbuf, inbuflen, EI_RPC_FETCH_STDOUT) < 0) {
```

Comment:
The return possible values of ei_rpc_to seem to be undocumented. Can you add a note about that in the docs?

Reply:
Good catch. I have added this to the documentation of ei_rpc_to and ei_xrpc_to